### PR TITLE
✨ Initialized React App and designed Login Page

### PR DESCRIPTION
**Pull request by harshgoel05**

### CODE REVIEW

File: client/src/components/button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+interface Props {
+  color: string;
+}
+
+const Button: React.FC<
+  Props | React.ButtonHTMLAttributes<HTMLButtonElement>
+> = (props) => {
+  const buttonStyles = {
+    background: props.color === "primary" ? " #1FA5FF" : "#f8f9fa",
+    color: props.color === "primary" ? "#FFFFFF" : "#000000",
```
Comment (harshgoel05) [inline client/src/components/button/Button.tsx:12]: color should be global

File: client/src/components/button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    borderRadius: "10px",
+    padding: "12px 40px",
```
Comment (harshgoel05) [inline client/src/components/button/Button.tsx:14]: always use padding spacing in multiple of 8 or 4

File: client/src/components/input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Input: React.FC<React.InputHTMLAttributes<HTMLInputElement>> = (
+  props
+) => {
+  const inputStyles = {
+    borderRadius: "8px",
+    padding: "4.5% 10%",
```
Comment (harshgoel05) [inline client/src/components/input/Input.tsx:8]: padding should be in pixels

File: client/src/components/input/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    width: "80%",
```
Comment (harshgoel05) [inline client/src/components/input/Input.tsx:9]: should be in pixels

File: client/src/components/input/Input.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    border: "none",
+    backgroundColor: "#e0ebfb",
```
Comment (harshgoel05) [inline client/src/components/input/Input.tsx:11]: hardcoded colors. make a constants for colors

File: client/src/hooks/useForm.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+
+function useForm<S>(
+  initial: S
```
Comment (harshgoel05) [inline client/src/hooks/useForm.ts:4]: what is S?

**Comment by PushkarDureja** [comment]
@harshgoel05  Please take a look

**Pull request merged**